scipy/constants/__init__.py

```diff
@@ -5,316 +5,104 @@
-``fluid_ounce``        one fluid ounce (US) in cubic meters
-``fluid_ounce_US``     one fluid ounce (US) in cubic meters
-``fluid_ounce_imp``    one fluid ounce (UK) in cubic meters
-``bbl``                one barrel in cubic meters
-``barrel``             one barrel in cubic meters
-===================    ========================================================
-
-Speed
------
-
-==================    ==========================================================
-``kmh``               kilometers per hour in meters per second
-``mph``               miles per hour in meters per second
-``mach``              one Mach (approx., at 15 C, 1 atm) in meters per second
-``speed_of_sound``    one Mach (approx., at 15 C, 1 atm) in meters per second
-``knot``              one knot in meters per second
-==================    ==========================================================
-
-
-Temperature
------------
-
-=====================  =======================================================
-``zero_Celsius``       zero of Celsius scale in Kelvin
-``degree_Fahrenheit``  one Fahrenheit (only differences) in Kelvins
-=====================  =======================================================
+List of all constants
+=====================
+The properties of the constants in :mod:`scipy.constants` are taken from
+[CODATA2018]_ and can be either accessed directly via
 
 .. autosummary::
    :toctree: generated/
+   
+   value      -- Value in physical_constants indexed by key
+   unit       -- Unit in physical_constants indexed by key
+   precision  -- Relative precision in physical_constants indexed by key
+   find       -- Return list of physical_constant keys with a given string
+   ConstantWarning -- Constant sought not in newest CODATA data set
 
-   convert_temperature
-
-Energy
-------
-
-====================  =======================================================
-``eV``                one electron volt in Joules
-``electron_volt``     one electron volt in Joules
-``calorie``           one calorie (thermochemical) in Joules
-``calorie_th``        one calorie (thermochemical) in Joules
-``calorie_IT``        one calorie (International Steam Table calorie, 1956) in Joules
-``erg``               one erg in Joules
-``Btu``               one British thermal unit (International Steam Table) in Joules
-``Btu_IT``            one British thermal unit (International Steam Table) in Joules
-``Btu_th``            one British thermal unit (thermochemical) in Joules
-``ton_TNT``           one ton of TNT in Joules
-====================  =======================================================
-
-Power
------
-
-====================  =======================================================
-``hp``                one horsepower in watts
-``horsepower``        one horsepower in watts
-====================  =======================================================
-
-Force
------
+or via the dictionaries stored in :mod:`scipy.constants.physical_constants`
```

Comment:
This is the cause of the doc build warnings, there is no module `physical_constants`. Shouldn't this just be `scipy.constants`?

Reply:
Ah no, you are wanting to refer to `physical_constants`, which is simply a dictionary not a module (so both the `:mod:` is wrong and the sentence may need a tweak:
```
In [3]: type(constants.physical_constants)                                                     
Out[3]: dict
```